### PR TITLE
monitor agent support

### PR DESCRIPTION
**Pull request by zapion**
support for mozMinion

### CODE REVIEW

**Comment by zapion** [comment]
@ShakoHo sample job config is showed below

{
  "job_info": {
    "pid": 9762,
    "program": "mtbf_job_runner.py"
  },
  "name": "mtbf_operation",
  "serial": "f07f41f8",
  "type": "TBD"
}

**Comment by ShakoHo** [comment]
{'name': "mtbf_operation",
            'output': {'dirpath': 'output',
                       'file': "unittest"},
            'serial': "2c0dc64c",
            'command': "",
            'job_info': {
                'jobname':'flamekk.vmaster.moztwlab01.512',
                'seriesname':'mtbf',
                'pid': 18845,
                'program': "chrome",
                'path': "/home/shako/PycharmProjects/mozMinions/conf/test.json",
                'host_name': "mtbf-10",
                'port_no': 8086,
                'user_name': "xxxxxx",
                'pwd': "xxxxxx",
                'database_name': "raptor"
            }}

**Comment by zapion** [comment]
updated.

File: mtbf_job_runner.py [hunk 1]
```diff
@@ -111,6 +119,16 @@ def change_memory(self):
         self.device_obj.create_adb_forward(self.port)
         return False
 
+    @action(enabled=False)
+    def config_raptor(self):
+        settings = self.settings
+        import pdb
```
Comment (ShakoHo) [inline mtbf_job_runner.py]: pdb ...XD

File: mtbf_job_runner.py [hunk 2]
```diff
@@ -232,6 +250,35 @@ def enable_certified_apps_debug(self):
...
+               'serial': self.serial,
+               'job_info': {'pid': os.getpid(),
+                            'program': sys.argv[0],
+                            }
+               }
+        if hasattr(self, 'raptor'):
+            raptor = self.raptor
+            job['job_info'].update(self.raptor)
+        import pdb
+        pdb.set_trace()
```
Comment (ShakoHo) [inline mtbf_job_runner.py]: pdb ...XD

**Comment by zapion** [comment]
updated.

**Comment by ShakoHo** [comment]
lgtm r+

**Pull request merged**